docker/configure_workers_and_start.py

```diff
@@ -532,14 +533,6 @@ def generate_worker_files(environ, config_path: str, data_dir: str):
         os.mkdir(log_dir)
 
 
-def start_supervisord():
-    """Starts up supervisord which then starts and monitors all other necessary processes
-
-    Raises: CalledProcessError if calling start.py return a non-zero exit code.
-    """
-    subprocess.run(["/usr/bin/supervisord"], stdin=subprocess.PIPE)
```

Comment:
inlining this and replacing with `os.exec` is just a driveby while I was passing. Avoiding the unnecessary subprocess makes signal handling more robust.